pegasus/sites.v3/code.org/views/ai_101_videos.haml

```diff
@@ -1,22 +1,25 @@
-- video_101_index = 6
+- video_101_index = 7
 
-- video_101_tags = [hoc_s(:module_label_new), hoc_s(:module_label_new), hoc_s(:module_label_coming_soon), hoc_s(:module_label_coming_soon), hoc_s(:module_label_coming_soon), hoc_s(:module_label_coming_soon)]
+- video_101_tags = [hoc_s(:module_label_available_now), hoc_s(:module_label_new), hoc_s(:module_label_new), hoc_s(:module_label_available_now), hoc_s(:module_label_coming_soon), hoc_s(:module_label_coming_soon), hoc_s(:module_label_coming_soon)]
 
-- video_101_title = [hoc_s(:ai_pl_101_video_title_1), hoc_s(:how_ai_works_hero_heading), hoc_s(:ai_pl_101_video_title_2), hoc_s(:ai_pl_101_video_title_3), hoc_s(:ai_pl_101_video_title_4), hoc_s(:ai_pl_101_video_title_5)]
+- video_101_title = [hoc_s(:ai_pl_101_video_title_1), hoc_s(:ai_pl_101_video_title_2), hoc_s(:ai_pl_101_llm_slides_title), hoc_s(:how_ai_works_hero_heading), hoc_s(:ai_pl_101_video_title_3), hoc_s(:ai_pl_101_video_title_4), hoc_s(:ai_pl_101_video_title_5)]
 
-- video_101_desc = [hoc_s(:ai_pl_101_video_desc_1), hoc_s(:ai_pl_101_video_desc_6), hoc_s(:ai_pl_101_video_desc_2), hoc_s(:ai_pl_101_video_desc_3), hoc_s(:ai_pl_101_video_desc_4), hoc_s(:ai_pl_101_video_desc_5)]
+- video_101_desc = [hoc_s(:ai_pl_101_video_desc_1), hoc_s(:ai_pl_101_video_desc_6), hoc_s(:ai_pl_101_llm_slides_desc), hoc_s(:ai_pl_101_video_desc_2), hoc_s(:ai_pl_101_video_desc_3), hoc_s(:ai_pl_101_video_desc_4), hoc_s(:ai_pl_101_video_desc_5)]
 
-- video_id = ["ai_101_01", "ai_chatbots"]
+- video_id = ["ai_101_01", "ai_101_02", "", "ai_chatbots"]
 
-- video_code = ["ik70A1ge-aY", "X-AWdfSFCHQ"]
+- video_code = ["ik70A1ge-aY", "SgGVlTTXxMI", "", "X-AWdfSFCHQ"]
```

Comment:
Is there a reason why there is an empty set of quotes in this list?

Reply:
Yep! That index item isn't a video, it's the Google slideshow below so I had to put a placeholder there to keep the loop working.

https://github.com/code-dot-org/code-dot-org/blob/90a4da7e704b6543a135c8289bfb2dd948b741d7/pegasus/sites.v3/code.org/views/ai_101_videos.haml#L22 

And same answer for [this comment](https://github.com/code-dot-org/code-dot-org/pull/53557#discussion_r1313314577).